### PR TITLE
js/ETH makeCompiledStdlib

### PR DESCRIPTION
**Pull request by DanBurton**
Note that this PR is cfx-beautiful-mv -> cfx-mv

### CODE REVIEW

**Comment by DanBurton** [comment]
ugh git did the wrong things on the rebase

File: js/stdlib/ts/module_typecheck.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// This file only exists to typecheck that modules satisfy an interface
```
Comment (jeapostrophe) [inline js/stdlib/ts/module_typecheck.ts:1]: This is cool!

**Review by jeapostrophe — APPROVED** [review]

**Comment by jeapostrophe** [comment]
You made it so that the merge target of this was cfx-mv, rather than master

**Comment by jeapostrophe** [comment]
I don't know how to switch it

**Pull request merged**